src-ocaml/tea_app.ml

```diff
@@ -177,6 +179,7 @@ let programLoop update view subscriptions initModel initCmd = function
       | Some _id ->
         let newVdom = [view !latestModel] in
         let justRenderedVdom = Vdom.patchVNodesIntoElement callbacks parentNode !priorRenderedVdom newVdom in
+        let () = renderCallback !latestModel in
```

Comment:
Yeah, this doesn't allow for any commands either, so a normal event might be a lot better.